recipes/vowpalwabbit/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: vowpalwabbit
+  version: 8.7.0
+
+source:
+  url: https://scgrahamarchive.blob.core.windows.net/vowpal-wabbit/vowpal_wabbit_2ee2bcf_rapidjson.tgz 
```

Comment:
Reminder to replace this with the upstream tarball (applying patches if required)

Reply:
@peterychang when can we get the tarball on pypi? should we hold on this till then?

Reply:
We just discussed the plan for the official pypi release. We should have the source tarball up either today or tomorrow, so its probably best to wait until its done